frontend/packages/erd-core/src/components/ERDRenderer/ERDContent/TableNode/type.ts

```diff
@@ -3,6 +3,7 @@ import type { Node } from '@xyflow/react'
 
 export type Data = {
   table: Table
+  isActiveHighlighted: boolean
```

Comment:
This property is not yet used.
I want to stop calling `useUserEditingStoreActive()` directly in TableNode, and refer to it from here.